sys-cluster/teleport/teleport-2.6.0.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit eutils golang-build systemd user
+
+DESCRIPTION="Modern SSH server for teams managing distributed infrastructure"
+HOMEPAGE="https://gravitational.com/teleport"
+
+EGO_PN="github.com/gravitational/${PN}/..."
+
+if [[ ${PV} == "9999" ]] ; then
+	inherit git-r3 golang-vcs
+	EGIT_REPO_URI="https://github.com/gravitational/${PN}.git"
+else
+	inherit golang-vcs-snapshot
+	SRC_URI="https://github.com/gravitational/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm"
+fi
+
+IUSE=""
+LICENSE="Apache-2.0"
+RESTRICT="test strip"
+SLOT="0"
+
+DEPEND="
+	app-arch/zip
+	>=dev-lang/go-1.9.2"
```

Comment:
Do not depend on go directly, wgere are you using zip?

Reply:
zip is used inside the Makefile for the package, to bundle the webassets in the main teleport binary: https://github.com/gravitational/teleport/blob/v2.6.0/Makefile#L156.

What's the recommended way to depend on go as a build dependency?

Reply:
go dependency provided by the eclass

Reply:
Fixed